application-workloads/jboss/jboss-eap-clustered-multivm-rhel/azuredeploy.json

```diff
@@ -350,7 +350,7 @@
               ]
             },
             "protectedSettings": {
-              "commandToExecute": "[format('sh jbosseap-setup-redhat.sh {0} {1} ''{2}'' {3} {4} ''{5}'' {6} {7} {8} {9} {10} {11}', variables('scriptArgs'), parameters('jbossEAPUserName'), parameters('jbossEAPPassword'), parameters('rhelOSLicenseType'), parameters('rhsmUserName'), parameters('rhsmPassword'), parameters('rhsmPoolEAP'), variables('storageAccountName'), variables('containerName'), base64(listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName')), '2021-02-01').keys[0].value), parameters('rhsmPoolRHEL'))]"
+              "commandToExecute": "[format('sh jbosseap-setup-redhat.sh {0} {1} ''{2}'' {3} {4} ''{5}'' {6} {7} {8} {9} {10}', variables('scriptArgs'), parameters('jbossEAPUserName'), parameters('jbossEAPPassword'), parameters('rhelOSLicenseType'), parameters('rhsmUserName'), parameters('rhsmPassword'), parameters('rhsmPoolEAP'), variables('storageAccountName'), variables('containerName'), base64(listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName')), '2021-02-01').keys[0].value), parameters('rhsmPoolRHEL'))]"
```

Comment:
you still have 11 args but the cmd is only accepting 10 - which is correct?

Reply:
@bmoore-msft Excluding the {0}, there are 10 arguments in this PR, I've tested out the same and it's working fine. Please approve the PRs.

Reply:
Hey @bmoore-msft, please approve the requested PRs, successful testing is done with expected results and is working fine.

Reply:
Het @bmoore-msft, following up on the previous comment, please approve the PRs.

Reply:
You have an extra argument in the function that's not being used... if it's not needed, remove it so the intent is clear.  If it is needed, use it.

Reply:
Hey @bmoore-msft, I've checked the number of arguments and all them are being used in the script; tested out the same & it's working just fine with the expected results. Please approve the PRs.